CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 18.2.0 (TODO)
+
+### React
+
+* Only treat updates to Suspended components as a new mount in Legacy Mode ([@rickhanlonii](https://github.com/rickhanlonii)) in [#24530](https://github.com/facebook/react/pull/24530)
```

Comment:
We don't use the term "legacy mode" in public communication much. Also this doesn't have any observable behavior (I think), it's just an internal refactor. So let's omit it?